New Fathead, see: https://duck.co/ideas/idea/515/languages-of-a-country

Roysten opened the pull request:
This is an attempt to add the Fathead as discussed here: https://duck.co/ideas/idea/515/languages-of-a-country. I have seen that someone else was also working on this Fathead (https://github.com/duckduckgo/zeroclickinfo-fathead/pull/54) but never finished it, so I decided to give it a try.

I'm not sure if the generated output.txt contains all the necessary information, so I have included a snippet:
```
1.Full article title	2.Type of article	3.For redirects only	4.Ignore.	5.Categories. 	6.Ignore.	7.Related topics. 	8.Ignore.	9.External links.	10.For disambiguation pages only. 	11.Image. 	12.Abstract. 	13.URL. 

Uzbekistan	A										Uzbek (official) 74.3%, Russian 14.2%, Tajik 4.4%, other 7.1%	https://www.cia.gov/library/publications/the-world-factbook/geos/uz.html
Vanuatu	A										local languages (more than 100) 63.2%, Bislama (official; creole) 33.7%, English (official) 2%, French (official) 0.6%, other 0.5% (2009 est.)	https://www.cia.gov/library/publications/the-world-factbook/geos/nh.html
Venezuela	A										Spanish (official), numerous indigenous dialects	https://www.cia.gov/library/publications/the-world-factbook/geos/ve.html
Vietnam	A										Vietnamese (official), English (increasingly favored as a second language), some French, Chinese, and Khmer, mountain area languages (Mon-Khmer and Malayo-Polynesian)	https://www.cia.gov/library/publications/the-world-factbook/geos/vm.html
Virgin Islands	A										English 71.6%, Spanish or Spanish Creole 17.2%, French or French Creole 8.6%, other 2.5% (2010 est.)	https://www.cia.gov/library/publications/the-world-factbook/geos/vq.html
Wallis and Futuna	A										Wallisian (indigenous Polynesian language) 58.9%, Futunian 30.1%, French (official) 10.8%, other 0.2% (2003 census)	https://www.cia.gov/library/publications/the-world-factbook/geos/wf.html
West Bank	A										Arabic, Hebrew (spoken by Israeli settlers and many Palestinians), English (widely understood)	https://www.cia.gov/library/publications/the-world-factbook/geos/we.html
Western Sahara	A										Standard Arabic (national), Hassaniya Arabic, Moroccan Arabic	https://www.cia.gov/library/publications/the-world-factbook/geos/wi.html
World	A										Mandarin Chinese 11.82%, Spanish 5.77%, English 4.67%, Hindi 3.62%, Arabic 3.3%, Portuguese 2.83%, Bengali 2.69%, Russian 2.33%, Japanese 1.7%, Javanese 1.15%, Standard German 1.09% (2014 est.)	https://www.cia.gov/library/publications/the-world-factbook/geos/xx.html
Yemen	A										Arabic (official)	https://www.cia.gov/library/publications/the-world-factbook/geos/ym.html
Zambia	A										Bembe 33.4%, Nyanja 14.7%, Tonga 11.4%, Lozi 5.5%, Chewa 4.5%, Nsenga 2.9%, Tumbuka 2.5%, Lunda (North Western) 1.9%, Kaonde 1.8%, Lala 1.8%, Lamba 1.8%, English (official) 1.7%, Luvale 1.5%, Mambwe 1.3%, Namwanga 1.2%, Lenje 1.1%, Bisa 1%, other 9.2%, unspecified 0.4%	https://www.cia.gov/library/publications/the-world-factbook/geos/za.html
Zimbabwe	A										Shona (official; most widely spoken), Ndebele (official, second most widely spoken), English (official; traditionally used for official business), 13 minority languages (official; includes Chewa, Chibarwe, Kalanga, Koisan, Nambya, Ndau, Shangani, sign language, Sotho, Tonga, Tswana, Venda, and Xhosa)	https://www.cia.gov/library/publications/the-world-factbook/geos/zi.html
The Bahamas	R	Bahamas, The									English (official), Creole (among Haitian immigrants)	https://www.cia.gov/library/publications/the-world-factbook/geos/bf.html
Bahamas	R	Bahamas, The									English (official), Creole (among Haitian immigrants)	https://www.cia.gov/library/publications/the-world-factbook/geos/bf.html
The Gambia	R	Gambia, The									English (official), Mandinka, Wolof, Fula, other indigenous vernaculars	https://www.cia.gov/library/publications/the-world-factbook/geos/ga.html
Gambia	R	Gambia, The									English (official), Mandinka, Wolof, Fula, other indigenous vernaculars	https://www.cia.gov/library/publications/the-world-factbook/geos/ga.html
```
Currently it ignores field 4 through 11. Also I'm not sure if I use field 3 correctly.
It adds variations for Gambia, The:
 _The Gambia_
_Gambia_ 
and Bahamas, The:
 _The Bahamas_
_Bahamas_.

https://duck.co/ia/view/country_languages

moollaza commented:
@Roysten thanks a lot for your contribution!

Can you please fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/pull_request_template_spice.md) and edit your initial comment. Once completed, we'll be sure to review your PR and give you some feedback.

Thanks!

Roysten commented:
# Fathead Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Show the language used in a country. For example:
_"language in United Kingdom"_ would show _"English"_

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Saves the user from navigating to a page and finding the same information after searching.

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://www.cia.gov/library/publications/the-world-factbook/fields/2098.html

**Why did you choose this data source?**
Contains all the necessary data, trustworthy source

**Are there any other alternative (better) data sources?**
Not that I'm aware

**What are some example queries that trigger this Instant Answer?**
language in Brazil
language in Mexico

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Students, travelers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
https://duck.co/ideas/idea/515/languages-of-a-country

**Which existing Instant Answers will this one supercede/overlap with?**
Any Instant Answer using countries as their trigger

**Are you having any problems? Do you need our help with anything?**
I'm not sure what fields in output.txt are required and where they will be used.
The reference field is unclear to me.

**Where did you hear about DuckDuckHack? (For first time contributors)**
Searched for DuckDuckGo Github, and clicked the link on the Github page

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
Since Duckpan cannot run Fatheads I have no image, but similar to other Fatheads I guess...

## Checklist
Please place an 'X' where appropriate.

```
[x] Added metadata and attribution information
[] Wrote test file and added to t/ directory
[] Verified that Instant Answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[] Verified that Instant Answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility
    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```
I see there is a styleguide, I will change my code to match the guide.

moollaza commented:
@Roysten thanks for updating! We'll be sure to review this shortly and give you some feedback!

jdorweiler commented on share/fathead/country_languages/fetch.sh:
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+mkdir download
+wget https://www.cia.gov/library/publications/the-world-factbook/fields/2098.html -O download/output.txt
```
I think this should be page.txt to fit with your parse.js script?

jdorweiler commented:
@Roysten looks great! 
![selection_083](https://cloud.githubusercontent.com/assets/1882892/8169469/c5eed98c-1377-11e5-9381-57942348b7c2.png)
https://ddh10.duckduckgo.com/?q=afghanistan+language&ia=about

Let me know if you think of any trigger words to add.

Roysten commented:
Thanks for your review. Looks great indeed! I fixed the error in fetch.sh (was testing locally and didn't upload the new version to codio.com). I can't really think of any additional trigger words...

jdorweiler commented:
@Roysten sorry I just noticed that I forgot to let you know we had some delays in adding new fatheads :(

This looks great though!  I'll merge and ping you when it's live.

jdorweiler commented:
@Roysten It's live!  https://duckduckgo.com/?q=afghanistan+language&ia=about 
Sorry for the long delay.
